src/Security/Member.php

```diff
@@ -529,8 +529,7 @@ public static function logged_in_session_exists()
     }
 
     /**
-     * @deprecated Use Security::setCurrentUser(null) or an IdentityStore
-     * Logs this member out.
+     * @deprecated Use Security::setCurrentUser(null) or an IdentityStore Logs this member out.
```

Comment:
```suggestion
     * @deprecated Use Security::setCurrentUser(null) or an IdentityStore instead
```
Also update the message and version in the notice below

Reply:
Updated